### ISSUE TITLE
 Unknown custom element: <font-awesome-icon> 

### ISSUE THREAD

**Issue by VadimCeretean** [nuxt-community/fontawesome-module#11]
Unknown custom element: font-awesome-icon - did you register the component correctly? For recursive components, make sure to provide the "name" option.

how can i fix it?

**Comment by Tjark-Kuehl** [comment]
I've got the same problem when I put the fontawesome configuration outside the buildModules item.
But when it's inside like so
```
[
	'@nuxtjs/fontawesome',
	{
		icons: {
			solid: ['faBars'],
			regular: [],
			light: [],
			duotone: [],
			brands: []
		}
	}
]
```
 it workes perfectly

**Comment by pimlie** [comment]
Thanks for the report, fixed in `v1.1.2`
